src/components/calcite-pick-list-group/calcite-pick-list-group.tsx

```diff
@@ -24,6 +25,11 @@ export class CalcitePickListGroup {
    */
   @Prop({ reflect: true }) groupTitle: string;
 
+  /**
+   * Number at which section headings should start for this component.
+   */
+  @Prop() headingLevel: HeadingLevel = 3;
```

Comment:
Would it be possible to set `level` on the parent and have it adjust and pass it to their children?

Reply:
We could, but that would add some complexity. I would say to hold off on this for now. 

For example, the `calcite-tip-manager` could add the `heading-level` to all of it's children `calcite-tip`s pretty easily since it already has an observer to query those. The `calcite-pick-list-group` is a little more complex because I think it can be nested so it would have to figure out it's heading from its parent.

Maybe separate issues for these?

Reply:
I'll take care of the tips one now but can you create an issue for the pick-list-group?

Reply:
Sure, I'll create follow-ups later today.